packages/assets/package.json

```diff
@@ -73,7 +73,7 @@
       [
         "@semantic-release/npm",
         {
-          "npmPublish": true
+          "npmPublish": false
         }
       ],
```

Comment:
Do we have to add the plugin at all if don't publish?

Reply:
I have left it in because in this case, it will not publish, but will at least adjust the `package.version` field.

Reply:
Doesn't do it without the package? :O

Reply:
semantic-release is unopinionated, we could also have a python project where everything is managed with pip. Think this is the reason you need the module at all. See https://www.npmjs.com/package/@semantic-release/npm#options